File: LoanManage.Web.Admin.UI/ClientApp/src/app/application/system-service/models/customFieldModel.ts
import { Injectable } from '@angular/core';

@Injectable()
export class CustomFieldModel
{
    id: number;
    customFieldName: string;
    customFieldCaption: string;
    customFieldControlTypeId: number;
    isCustomFieldMandatory: boolean;
    customFieldGroupName: string;
    customFieldSelectionValue: string;
    groupSortOrder: number;
    controlSortOrder: number;
}

File: LoanManagerAPI_V2/LoanManager.ApplicationService/Models/CustomFieldModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace LoanManager.ApplicationService.Models
{
    public class CustomFieldModel
    {
        public long Id { get; set; }
        public string CustomFieldName { get; set; }
        public string CustomFieldCaption { get; set; }
        public long CustomFieldControlTypeId { get; set; }
        public bool IsCustomFieldMandatory { get; set; }
        public string CustomFieldGroupName { get; set; }
        public string CustomFieldSelectionValue { get; set; }
        public int GroupSortOrder { get; set; }        
        public int ControlSortOrder { get; set; }
        
    }
}
